src/openforms/scss/admin/_app_overrides.scss

```diff
@@ -202,6 +202,15 @@ $djai-border-width: 8px;
     color: #999; // same color as django forms.css
   }
 
+  input#root_objecttype.form-control,
+  input#root_productaanvraagType.form-control,
+  input#root_informatieobjecttypeSubmissionReport.form-control,
+  input#root_informatieobjecttypeSubmissionCsv.form-control,
+  input#root_informatieobjecttypeAttachment.form-control,
+  input#root_organisatieRsin.form-control {
+    width: 25em;
+  }
```

Comment:
generally we avoid ID-based and HTML element-based selectors, but instead we favour classnames only

Can you check if we can leverage the `uiSchema` feature of [react-json-schema-form](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/uiSchema/)? This is hooked up from serializer data here: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/form_design/registrations/index.js#L10